src/main/java/net/huskycraft/blockyarena/games/Game.java

```diff
@@ -144,12 +143,17 @@ private void inspect() {
                 teamB.add(gamersItr.next());
             }
             gameState = GameState.STARTING;
-            startingCountdown(10);
+            startingCountdown(BlockyArena.getInstance().getRootNode().getNode("timers", "lobby", "cooldownSec").getInt());
         } else if (timer != null) {
             timer.cancel();
             gameState = GameState.RECRUITING;
             broadcast(Text.of("Waiting for more players to join ..."));
+            
         }
+        else {
+        	MessageChannel.TO_ALL.send((Text)Text.builder("New arena game ! join with /ba join solo !").color(TextColors.GOLD).build());
```

Comment:
I love the idea of broadcasting new available game session to other players. However, we might need to deal with 2v2 so the message needed be adjusted accordingly (`/ba join doubles`).